concepts/basics/about.md

```diff
@@ -20,6 +20,12 @@ In the examples below you'll encounter:
   It it extremely useful for humans to view the data when it's nicely formatted.
   However that's not necessary for machines: the `--compact-output` option removes the formatting whitespace to minimize the size of the resulting JSON.
 
+- `-f filename` or `--from-file filename`
+
+  Instead of providing the `jq` program on the command line, read it from `filename.
```

Comment:
```suggestion
  Read the `jq` program from `filename` instead of reading it from the command line argument.
```

Starting a sentence with "Instead" tells the reader what is *not* the case. Prefer telling the reader what *is* the case.